anesthetic/samples.py

```diff
@@ -388,14 +393,22 @@ def plot_2d(self, axes=None, *args, **kwargs):
                     if x in self and y in self and lkwargs['kind'] is not None:
                         xlabel = self.get_label(x)
                         ylabel = self.get_label(y)
+                        if ~np.isfinite(self[x]).all():
+                            warnings.warn(f"column {x} has inf values.")
```

Comment:
This will currently warn that there are inf values when possibly there are no inf values, just some nan values, which from the pandas perspective are fine to be ignored... 

Might be better to work with `np.isinf(self[x]).any()`, here.

Reply:
done in https://github.com/handley-lab/anesthetic/pull/324/commits/a498a0b1304b26b627b3ce2732200c7258b80a6c